packages/contentstack-bulk-publish/src/util/client.js

```diff
@@ -1,11 +1,27 @@
-const { managementSDKClient } = require('@contentstack/cli-utilities');
+const { managementSDKClient, isAuthenticated } = require('@contentstack/cli-utilities');
 const { Command } = require('@contentstack/cli-command');
 const command = new Command();
 
 async function getStack(data) {
-  const tokenDetails = command.getToken(data.alias);
-  const managementClient = await managementSDKClient({ host: data.host, branchName: data.branch });
-  const stack = managementClient.stack({ api_key: tokenDetails.apiKey, management_token: tokenDetails.token });
+  const options = {
+    host: data.host,
+    branchName: data.branch,
+  };
+  const stackOptions = {};
+  if (data.alias) {
+    const tokenDetails = command.getToken(data.alias);
+    options.management_token = tokenDetails.token;
+    stackOptions.management_token = tokenDetails.token;
+    stackOptions.api_key = tokenDetails.apiKey;
+  }
+  if (data.stackApiKey) {
```

Comment:
@vkalta Can you check stack-api-key condition in else if?
If someone passed alias and stack-api-key both the stack-api-key from the management token stack will be overridden by the below stack-api-key.